dynamiqs/integrators/mesolve/rouchon_integrator.py

```diff
@@ -31,7 +31,7 @@ def prod(self, vf, control):
         pass
 
 
-class RouchonSolver(dx.AbstractSolver):
+class RouchonDXSolver(dx.AbstractSolver):
```

Comment:
Note: this inherits from `dx.AbstractSolver` so I classified this as a `DXSolver` (another object separate from solvers or from integrators)